src/handlers/usersHandlers.ts

```diff
@@ -22,18 +21,14 @@ const getUsers = (
 		return;
 	}
 	const paginationParameters = extractPaginationParameters(req);
-	const searchParameters = extractSearchParameters(req);
-	const authenticationIdParameters = extractAuthenticationIdParameters(req);
+	const { offset, limit } = getLimitValues(paginationParameters);
+	const { authenticationId } = extractAuthenticationIdParameters(req);
 
 	(async () => {
-		const userBundle = await loadUserBundle(
-			{
-				...getLimitValues(paginationParameters),
-				...searchParameters,
```

Comment:
Just wanted to point out that we were incorrectly passing `searchParameters` to a function that did not use or expect them.  This new structure won't actually prevent this mistake in future, but it *WILL* mean those values don't get further passed along to the query builder.

Reply:
Good catch!